Externals/glslang/glslang/Include/InfoSink.h

```diff
@@ -75,9 +75,9 @@ class TInfoSinkBase {
     TInfoSinkBase& operator<<(int n)                   { append(String(n)); return *this; }
     TInfoSinkBase& operator<<(unsigned int n)          { append(String(n)); return *this; }
     TInfoSinkBase& operator<<(long unsigned int n)     { append(String(n)); return *this; }
-    TInfoSinkBase& operator<<(float n)                 { const int size = 40; char buf[size]; 
-                                                         snprintf(buf, size, (fabs(n) > 1e-8 && fabs(n) < 1e8) || n == 0.0f ? "%f" : "%g", n);
```

Comment:
this check is unreliable, though I didn't fix many `-Wfloat-equal` warnings, this one seemed easy